lms/djangoapps/certificates/api.py

```diff
@@ -234,8 +235,9 @@ def certificate_downloadable_status(student, course_key):
         'download_url': None,
         'uuid': None,
     }
+    may_view_certificate = get_course_by_id(course_key).may_certify()
```

Comment:
I think there was something on one of Alex's PRs about using course overview instead of this to avoid hitting the modulestore, I'll find it.

Reply:
https://github.com/edx/edx-platform/pull/15937/files#diff-0bc1f107306323c502ed1fed39be2ddeR32

Reply:
Nice